### ISSUE TITLE
Add Signature Field

### ISSUE THREAD

**Issue by TSeine** [verbb/formie#203]
**What are you trying to do?**
A description of what you want to happen.
Be able to add a signature field

**What's your proposed solution?**
A description of how you think this could be solved, including any alternatives that you considered.

**Additional context**

**Comment by engram-design** [comment]
Can you provide details on what you mean by a signature field? As in, you want to be able to accept someone writing their signature in a field?

**Comment by TSeine** [comment]
Yes that is exactly what I meant.  For example - signing off on a work order, or purchase order, or delivery received notice. 

Trina
Sent from iPad

On Nov 3, 2020, at 4:10 PM, Josh Crawford <notifications@github.com> wrote:

﻿
Can you provide details on what you mean by a signature field? As in, you want to be able to accept someone writing their signature in a field?

**Comment by engram-design** [comment]
I'll give this some thought. My concerns are really around usability on desktop, and this being a very specialised field that's not common to most sites.

**Comment by TSeine** [comment]
Ok sounds good. Thx. Prior to my developer switching me over to Craft CMS, I used Wordpress and Jotform to build workplace safety forms - https://www.jotform.com/ 


Trina
Sent from iPad

On Nov 3, 2020, at 4:34 PM, Josh Crawford <notifications@github.com> wrote:

﻿
I'll give this some thought. My concerns are really around usability on desktop, and this being a very specialised field that's not common to most sites.

**Comment by RitterKnightCreative** [comment]
+1 for this. Have a client who I inherited using cognito forms now but we'd probably switch them over to Formie if this were available. Might be useful for client onboarding as well.

They have a nice [demo](https://www.cognitoforms.com/features/electronic-signatures) on the site which is how I'd think it could work in Formie.  IIIRC Gravity Forms for WP has this as well.

I had the same concern about drawing on a desktop but it's more usable than I thought... we just refinanced our mortgage awhile back and it was all electronic too. Obviously Formie wouldn't take the place of something like DocuSign but it could be a nice stand in for less "formal" applications...
